en/advanced_features/satcom.md

```diff
@@ -0,0 +1,221 @@
+# Iridium/RockBlock Satellite Communication System
+A satellite communication system can be used to provide long range high latency link between a ground station and a vehicle.
+
+This topic describes how to set up a system that uses RockBlock as the service provider for the Iridium SBD Satellite Communication System. Given good signal quality, users can expect a latency between 10 to 15 seconds.
+
+## Overview
+The following components are required for an operating satellite communication link:
+* A [RockBlock 9603](http://www.rock7mobile.com/products-rockblock-9603) module connected to a Pixhawk flashed with the PX4 Autopilot.
+* A message relay server running Ubuntu.
+* A ground station computer with QGroundControl installed and running Ubuntu 14.04 or 16.04 as the operating system.
+
+The link running cost consists of a line rental and per message cost:
+* Each module needs to be activated which costs £10.00 per month
+* Each message transmitted over the system costs one Credit per 50 bytes. Bundles of Credits can be bought from RockBlock for £0.04-£0.11 per Credit depending on the bundle size.
+
+Notes: 
+* It might be possible to run the system also on other ground station OS but this has not been tested yet and therefore is not guaranteed to work.
+* The [RockBlock MK2](http://www.rock7mobile.com/products-rockblock) can also be used but the RockBlock 9603 module is recommended as it is smaller and lighter while providing the same functionality.
+* Refer to the [RockBlock Documentation](https://docs.rockblock.rock7.com/docs) for a detailed explanation of the modules, running costs and RockBlock in general.
+
+In the following figure the full system architecture as implemented in the PX4 Autopilot is shown:
+
+![Architecture](../../assets/satcom/architecture.jpg)
+
+## Set-up
+
+In this section the set-up of the individual system components is described.
+
+#### Vehicle
+###### Wiring
+Connect the RockBlock module to a serial port of the Pixhawk. Due to the power requirements of the module it can only be powered over a high-power serial port as a maximum of 0.5 A at 5 V are required. If none is available/free then another power source which has the same ground level as the Pixhawk and can provide required power has to be setup. The details of the [connectors](https://docs.rockblock.rock7.com/docs/connectors) and the [power requirements](https://docs.rockblock.rock7.com/docs/power-supply) can be found in the RockBlock documentation.
```

Comment:
Do we have documentation which UARTs are high-power?

Reply:
The only info I found was in the [ardupilot telemetry doc](http://ardupilot.org/copter/docs/common-telemetry-port-setup-for-apm-px4-and-pixhawk.html) but I am not sure if that is true for all boards. In general I also think that Telem1 is already used for the regular telemetry.

Reply:
Hmm I don't know either. The APM docs only mention Telem1. Do you use it on that port?
@davids5 do you know about the maximum Amps for UART ports on Pixhawks?

Reply:
No, we are using the Serial6 and power it from an external source.

Reply:
@bkueng Clasicaly TELEM1 has the "High Power" supply.

FMUv2- BQ24315 Powers SERIAL1 (TELEM1) as VDD_5V_HIPOWER  1.5A max. It is controlled and has over current sensing.

FMUv4 - has a 1 A fuse. It is not controlled and has no over current sensing.

FMUv4pro - BQ24315 Powers TELEM1 as VDD_5V_HIPOWER  1.0A max.  It is controlled and has over current sensing.

FMUv5 HW - BQ24315 Powers TELEM1 as VDD_5V_HIPOWER  1.5A max.  It is controlled and has over current sensing.